docs/en_US/reference/experiment_config.rst

```diff
@@ -638,15 +676,15 @@ default: ``False``
 
 
 openpaiConfig
```

Comment:
Because OpenPai training service has a config field named ``openpaiConfig``, and the headers are not case sensitive. Just use ``openpaiConfig`` can not link to the correct section, and old anchor is invalid, so I use ``:ref``

Reply:
got it, thx.